### PR TITLE
chore: initialize web-frontend with next.js

### PR DESCRIPTION
**Pull request by salet-gutierrez-jalafund**
### What did I do?
I initialized the program for the frontend side with the chosen technology of next.js

### Why did I do it?
So that the group could start developing the view.

### How did I do it?
I followed the steps to run a nextjs command for a website and according to the notes and acceptance criteria that are in the US

[folder structure](https://github.com/luiggy-mamani-jalafund/DDD-CinemaReservationSystem/wiki/Folder-structure)

### CODE REVIEW

**Comment by salet-gutierrez-jalafund** [comment]
[ID-02: Set Up Frontend Environment with Next.js](https://trello.com/c/3FzjRXLE/4-id-02-set-up-frontend-environment-with-nextjs)

File: web-frontend/src/app/styles/globals.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+$background-light: #ffffff;
```
Comment (luiggy-mamani-jalafund) [inline web-frontend/src/app/styles/globals.scss]: I guess this is the reset css file, so please move it to the base folder

**Review by luiggy-mamani-jalafund — CHANGES_REQUESTED** [review]
please move the fonts directory to the public directory

**Comment by salet-gutierrez-jalafund** [comment]
> please move the fonts directory to the public directory

In the folder structure that is within the wiki ([folder structure](https://github.com/luiggy-mamani-jalafund/DDD-CinemaReservationSystem/wiki/Folder-structure)) it is explained why the fonts are being entered into that folder.

**Comment by luiggy-mamani-jalafund** [comment]
> > please move the fonts directory to the public directory
> 
> In the folder structure that is within the wiki ([folder structure](https://github.com/luiggy-mamani-jalafund/DDD-CinemaReservationSystem/wiki/Folder-structure)) it is explained why the fonts are being entered into that folder.

The public folder stores public files such as fonts, and CSS fonts must be configured to use them.

Screenshot from 2024-09-25 10-33-14

**Comment by luiggy-mamani-jalafund** [comment]
please move the components, pages, services, and styles at the same height of the app.

Also, move the public folder to the same height of the src

**Review by luiggy-mamani-jalafund — APPROVED** [review]

**Pull request merged**